test/run_notebook.sh

```diff
@@ -38,6 +38,7 @@ nbs_2=(
   'sampler_study.ipynb'
   'sampling_diagnostics.ipynb'
   'model_selection.ipynb'
+  'bayes_factors.ipynb'
```

Comment:
Fine as-is but, since it's about many methods for evidence computation, maybe `model_evidence_and_bayes_factors.ipynb`